src/test/fuzz/connman.cpp

```diff
@@ -104,7 +104,9 @@ FUZZ_TARGET_INIT(connman, initialize_connman)
             },
             [&] {
                 // Limit now to int32_t to avoid signed integer overflow
-                (void)connman.PoissonNextSendInbound(fuzzed_data_provider.ConsumeIntegral<int32_t>(), fuzzed_data_provider.ConsumeIntegral<int>());
+                (void)connman.PoissonNextSendInbound(
+                        std::chrono::microseconds{fuzzed_data_provider.ConsumeIntegral<int32_t>()},
+                        std::chrono::microseconds{fuzzed_data_provider.ConsumeIntegral<int>()});
```

Comment:
The meaning of the second argument was "seconds" before, to keep the behavior:

```suggestion
                (void)connman.PoissonNextSendInbound(
                        std::chrono::microseconds{fuzzed_data_provider.ConsumeIntegral<int32_t>()},
                        std::chrono::seconds{fuzzed_data_provider.ConsumeIntegral<int>()});
```

Reply:
I think it is fine to change the meaning with the benefit that the fuzz test will be also grinding at a smaller resolution, which is more likely to be used. I don't think we have a poisson delay of int::max() seconds anywhere. And I don't think it makes sense in practice.

Reply:
Left it as-is.